xmodule/static/css-builtin-blocks/SequenceBlockDisplay.css

```
@@ -78,6 +78,8 @@
     position: relative;
     height: 100%;
     flex-grow: 1;
+    /*overflow-x: auto;*/
+    max-width: 832px;
```

```suggestion
    max-width: calc(100% - 80px);
```
The bar was getting rendered outside the Iframe sometimes on small screen sizes. With this change, the unit buttons take the (full width of parent - 2 nav buttons) which have a max-width of 40px.